include/circt/Dialect/SV/SVOps.h

```diff
@@ -20,6 +20,9 @@
 #include "mlir/Interfaces/SideEffectInterfaces.h"
 
 namespace circt {
+namespace hw {
+  class InstanceOp;
+}
```

Comment:
Should this be forward declared here in the header or should this be something in `SVOps.cpp`? I don't know what is the good approach here, but I've only done this when I actually need this in `*.h` and not when I need it in `*.cpp`.

I may be missing something here.

Reply:
I was in some checking code from tablegen, so it needed to be in the header.  I'll double check.

Reply:
has to be here.